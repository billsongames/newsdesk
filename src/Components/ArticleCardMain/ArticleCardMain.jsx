import React from "react";
import PropTypes from "prop-types"

function ArticleCardMain(props) {
  const {title, description, content, url, image, source, publishedAt} = props

  return(
    <div>
      <div>
        {title}
      </div>

    </div>
  )
  
}

ArticleCardMain.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  content: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  source: PropTypes.string.isRequired,
  publishedAt: PropTypes.string.isRequired
}

export default ArticleCardMain