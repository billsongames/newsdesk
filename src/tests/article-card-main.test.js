import React from "react";
import { render } from "@testing-library/react";
import ArticleCardMain from "../Components/ArticleCardMain/ArticleCardMain";

describe('Name of the group', () => {
  const validProps = {
    title: "test title", 
  }

  it('renders correctly', () => {
    const { asFragment } = render(<ArticleCardMain 
      title={validProps.title} 
    />);

    expect(asFragment()).toMatchSnapshot();
  });
});


