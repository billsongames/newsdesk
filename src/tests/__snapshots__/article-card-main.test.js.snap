// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name of the group renders correctly 1`] = `
<DocumentFragment>
  <div>
    <div>
      test title
    </div>
  </div>
</DocumentFragment>
`;
